tests/test_cmd2.py

```diff
@@ -1860,22 +1860,6 @@ def test_alias_create(base_app, capsys):
     out = run_cmd(base_app, 'alias list fake')
     assert out == normalize('alias create fake pyscript')
 
-def test_alias_quoted_name(base_app, capsys):
```

Comment:
Somehow removing these unit tests and/or changing code elsewhere in this PR has resulted in an uncovered line in **parsing.py** when ``_is_valid_command()`` is called with an empty string.  Please add a simple unit test for this other case.  I'd like to keep the coverage of **parsing.py** at 100% given its critical underlying role.

Reply:
Done